src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -15,20 +16,30 @@ namespace Microsoft.ML.Runtime.Data
     public sealed class EstimatorChain<TLastTransformer> : IEstimator<TransformerChain<TLastTransformer>>
         where TLastTransformer : class, ITransformer
     {
+        // Host is not null iff there is any 'true' values in _needCacheAfter (in this case, we need to create an instance of
```

Comment:
>iff [](start = 28, length = 3)

typo

Reply:
No, 'iff' means 'if and only if'

---
In reply to: [231778264](https://github.com/dotnet/machinelearning/pull/1569#discussion_r231778264) [](ancestors = 231778264)